composeApp/src/desktopMain/kotlin/edu/dyds/movies/data/local/LocalData.kt

```diff
@@ -0,0 +1,17 @@
+package edu.dyds.movies.data.local
+
+import edu.dyds.movies.domain.entity.RemoteMovie
+
+class LocalData: LocalDataSource {
+    private val moviesCache: MutableList<RemoteMovie> = mutableListOf()
```

Comment:
Debería trabajar con Movie no con RemoteMovie. Razon: La cache no debería depender de los datos crudos de la API remota
```suggestion
    private val moviesCache: MutableList<Movie> = mutableListOf()
```
Este cambio va a impactar en los servicios de la interfaz también